pkg/network/setup/podnic.go

```diff
@@ -96,7 +96,11 @@ func newPhase2PodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, handle
 	if err != nil {
 		return nil, err
 	}
-	podnic.podInterfaceName = ifaceLink.Attrs().Name
+	if ifaceLink == nil {
+		podnic.podInterfaceName = ""
+	} else {
+		podnic.podInterfaceName = ifaceLink.Attrs().Name
+	}
```

Comment:
nit:
```suggestion
if ifaceLink != nil {
    podnic.podInterfaceName = ifaceLink.Attrs().Name
}
```

Reply:
I prefer to keep it explicit, so it will be visible the `podnic.podInterfaceName` can be "".